samples/highcharts/chartchooser/continuous-composition-stacked-area-monochrome/demo.js

```diff
@@ -0,0 +1,70 @@
+Highcharts.chart("container", {
+    chart: {
+        type: "areaspline"
+    },
+    title: {
+        text: "Msis atmospheric composition by height"
+    },
+    subtitle: {
+        text:
+        'Source: <a href="https://en.wikipedia.org/wiki/Atmosphere_of_Earth" target="_blank">Wikipedia.org</a>'
+    },
+    xAxis: {
+        tickmarkPlacement: "on",
+        title: {
+            text: "Height (km)"
+        },
+        labels: {
+            formatter: function () {
+                return this.value * 100;
```

Comment:
Wouldn't it be better to have the data be correct so that we don't have to hack the labels? It can be done by setting `plotOptions.areasplie.pointInterval` to 100.

Reply:
Done 👍